test/test_chordpro/test_parser.py

```diff
@@ -15,17 +15,13 @@
 from .. import disable_logging
 from .. import dynamic # pylint: disable=unused-import
 
-LANGUAGES = {
-    'tex': 'latex',
-    'sgc': 'chordpro',
-    'html': 'html',
-}
+OUTPUTS = ['csg', 'tsg', 'html']
 
 class FileTest(unittest.TestCase, metaclass=dynamic.DynamicTest):
     """Test of chorpro parser, and several renderers.
 
     For any given `foo.source`, it is parsed as a chordpro file, and should be
-    rendered as `foo.sgc` with the chordpro renderer, and `foo.tex` with the
+    rendered as `foo.csg` with the chordpro renderer, and `foo.tsg` with the
     latex renderer.
```

Comment:
On peut ajouter `and so on` pour dire qu'on vérifie aussi la conversion vers du HTML, et vers n'importe quel format qui sera ajouté plus tard…